### PR TITLE
[TASK] Make TileView functional

### LINKED ISSUE #27: Tile View Component
grafik

The tile view contains files and folders in three groups when a parent storage is selected (see screen):
- Folders
- Files
- Images

The tile itself should be generic and react only to data given.

The tiles view has different views, corresponding to its state:
Empty:
grafik

Empty/Read-only:
grafik

Storage Selection:
grafik

### PR DESCRIPTION
**Pull request by NeoBlack**
Fix #27

### CODE REVIEW

File: Build/Vue/src/components/Tile/index.tsx [hunk 1]
```diff
@@ -20,49 +20,121 @@ export default class Tile extends Vue {
...
+
     constructor(props: any) {
         super(props);
     }
 
     private render(h: CreateElement): VNode {
-        // fix me, I'm ugly
-        let classes = 'panel panel-default tile ' + this.$props.type;
+        if (this.type === 'FOLDER') {
+            return this.renderFolder(h);
```
Comment (susannemoog) [inline Build/Vue/src/components/Tile/index.tsx]: you should be able to remove all references to h / CreateElement - that's globally mapped in latest versions of vuejs/jsx

Comment (NeoBlack) [reply Build/Vue/src/components/Tile/index.tsx]: good point, I saw it but forget to remove it

**Comment by andreasfernandez** [comment]
I get a console warning:

[Vue warn]: The computed property "readOnly" is already defined in data.

**Comment by NeoBlack** [comment]
> I get a console warning:
> 
> [Vue warn]: The computed property "readOnly" is already defined in data.

thx, fixed

**Comment by NeoBlack** [comment]
+1

**Pull request merged**